bootable-jar/runtime/src/main/java/org/wildfly/core/jar/runtime/_private/BootableJarLogger.java

```diff
@@ -165,4 +165,8 @@ public interface BootableJarLogger extends BasicLogger {
 
     @Message(id = Message.NONE, value = "Path to a CLI script to execute when starting the Bootable JAR")
     String argCliScript();
+
+    @LogMessage(level = DEBUG)
+    @Message(id = 25, value = "Failed to initialize a security provider. Exception: %s")
```

Comment:
Minor nit, but should the last word be "Reason" instead of "Exception"?

Reply:
Good! Fixed.